src/taxman.py

```diff
@@ -137,7 +137,14 @@ def _evaluate_taxation_GERMANY(
                         f"({sc.op.__class__.__name__})"
                         for sc in sold_coins
                     )
-                    tx = transaction.TaxEvent(taxation_type, taxed_gain, op, remark)
+                    tx = transaction.TaxEvent(
+                        taxation_type,
+                        taxed_gain,
+                        op,
+                        total_win,
+                        total_win - taxed_gain,
```

Comment:
Might be enough to save only `total_win` in `TaxEvent` as we can calculate this on the fly.

Reply:
Should have become irrelevant now due to removal of buy price column.